bundle/manifests/bpfman-operator.clusterserviceversion.yaml

```diff
@@ -335,7 +335,7 @@ metadata:
     operatorframework.io/arch.arm64: supported
     operatorframework.io/arch.ppc64le: supported
     operatorframework.io/arch.s390x: supported
-  name: bpfman-operator.v0.5.2
+  name: bpfman-operator.v0.5.4
```

Comment:
What about all the examples?
```
                "url": "quay.io/bpfman-bytecode/go-app-counter:latest"
                "url": "quay.io/bpfman-bytecode/fentry:latest"
                "url": "quay.io/bpfman-bytecode/fexit:latest"
                "url": "quay.io/bpfman-bytecode/kprobe:latest"
                "url": "quay.io/bpfman-bytecode/tc_pass:latest"
                "url": "quay.io/bpfman-bytecode/tracepoint:latest"
                "url": "quay.io/bpfman-bytecode/uprobe:latest"
                "url": "quay.io/bpfman-bytecode/xdp_pass:latest"
    containerImage: quay.io/bpfman/bpfman-operator:latest
                image: quay.io/bpfman/bpfman-operator:latest
```

Reply:
I can't change those until after the release pr merges because `make bundle` resets them all, and there's a check to make sure whatever you check in matches what `make bundle` generates.  I'm not sure about the first item you mentioned.  In any case, all those things need to change before we push the bundles to the community sites.

It seems to me that `make bundle` should update ALL the version numbers that need to be updated, but that's a future issue.

Reply:
Just to confirm, [this check](https://github.com/bpfman/bpfman-operator/blob/main/.github/workflows/pull_request.yml#L74) will fail if I edit the CSV after running `make bundle`.